tests/test_match_labels.py

```diff
@@ -15,3 +15,13 @@ def test_match_labels():
     
     assert(all(match))
 
+
+def test_match_labels_2():
+    labels_x = np.asarray([1, 1, 1, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
+    labels_y = np.asarray([1, 1, 2, 0, 0, 3, 3, 4, 4, 4, 0], dtype=np.uint8)
+
+    reference_y_matched = np.asarray([1, 1, 5, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
```

Comment:
Hi Johannes @jo-mueller 

could you please take a look at that test? I think this should fail. I think, the "5" should be a "1" in the reference. What do you think?

Reply:
Yes, `reference_y_matched ` should essentially be a copy of `labels_x ` here.

Reply:
A shorter version of this test could be:

```python
labels_x = np.asarray([1, 1, 1, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
labels_y = np.asarray([1, 1, 2, 0, 0, 3, 3, 4, 4, 4, 0], dtype=np.uint8)

assert np.array_equal(labels_x, bputils.label.match_labels(labels_x, labels_y))
```

Reply:
Yes, that's true. I changed that in fd0f669ea1ad8a6510c57f56998ae4f605dc1f9b. Now there are two tests for the label matching - is this necessary or does the 2nd test cover something I overlooked?

Reply:
Yes, no I got it! Thanks :-)